OmniBLE/Bluetooth/Pair/KeyExchange.swift

```diff
@@ -27,16 +27,16 @@ class KeyExchange {
     var pdmConf: Data
     var ltk: Data
     
```

Comment:
Use an interface for the Private key. That will allow us to mock key generators in unit tests.